src/components/structural/header/Header.js

```diff
@@ -1,707 +1,707 @@
-import React, { Component, Fragment } from 'react';
```

Comment:
This shouldn't have had any changes. Can you see what's up with this diff?

Reply:
I'm suspecting that this might be able to be fixed by running the VSCode autoformatter. That often fixes this type of issue.

Reply:
I run autoformatter in VSCode and I don't see any changes in Header.js file.

Reply:
Did you delete the header file at all and then re-add it?

This is part of the output of the diff.
```--- a/src/components/structural/header/Header.js
+++ b/src/components/structural/header/Header.js```

Reply:
No, I didn't delete the all Header.js file but the part where I add the code in the first commit.

Reply:
You might need to reset and recreate the commit. I'm not sure what's going on with that.

Reply:
I made a new branch with the exact same change and it doesn't highlight anything from header.js. Can I create a new pull request base of that branch?

Reply:
Yes